pkgs/tools/system/cm-rgb/default.nix

```diff
@@ -32,8 +34,10 @@ buildPythonApplication rec {
   propagatedBuildInputs = [
     click
     hidapi
+    lm_sensors
     psutil
     pygobject3
+    pysensors
```

Comment:
Are you sure that this is necessary? All the temperature options work fine here. E.g.:

```
cm-rgb-monitor --list-temp-sources
Available sensors:
nvme/Composite  ->  40.85
k10temp/Tctl  ->  70.25
k10temp/Tdie  ->  70.25
k10temp/Tccd1  ->  68.5
Available fan speed sensors for mirage:
```

`temp-source` and `show-temp` also work fine.

From a quick glance of the cm-rgb source, I could also not find a reference to PySensors. They seem to be using `psutil` for everything.

Reply:
I've just checked the source. I agree `PySensors` is not required. I just looked at the docs and assumed I need the dependence and never tested it without.